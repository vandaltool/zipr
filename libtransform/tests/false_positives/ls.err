*** PEASOUP: Add/Sub overflow detected at: 0x804ff27 nextPC:0xf0000dd0 esp:0xbfe4b4cc eax:0x1a ebx:0xbfe4b3d0 ecx:0xffffffff edx:0xbfe4b574 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x804bc71 nextPC:0xf0001970 esp:0xbfe4b3cc eax:0x9258328 ebx:0xbfe4b3a0 ecx:0x77 edx:0xbfe4b57c POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x804bc71 nextPC:0xf0001970 esp:0xbfe4b06c eax:0x9258328 ebx:0xbfe4b040 ecx:0x77 edx:0xb7718694 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055130 nextPC:0xf0001f10 esp:0xbfe4af4c eax:0xb70d981c ebx:0xbfe4af00 ecx:0xb70d9814 edx:0xb70d9810 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055165 nextPC:0xf0001f30 esp:0xbfe4af9c eax:0xbfe4adf0 ebx:0xbfe4af50 ecx:0x3 edx:0xb70d9804 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055130 nextPC:0xf0001f10 esp:0xbfe4af9c eax:0xb70d981c ebx:0xbfe4af50 ecx:0xb70d97fc edx:0xb70d97f8 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055130 nextPC:0xf0001f10 esp:0xbfe4af9c eax:0xb70d981c ebx:0xbfe4af50 ecx:0xb70d97f0 edx:0xb70d97ec POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055130 nextPC:0xf0001f10 esp:0xbfe4af9c eax:0xb70d981c ebx:0xbfe4af50 ecx:0xb70d97e4 edx:0xb70d97e0 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055165 nextPC:0xf0001f30 esp:0xbfe4afec eax:0xbfe4ae40 ebx:0xbfe4afa0 ecx:0x3 edx:0xb70d97d4 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055130 nextPC:0xf0001f10 esp:0xbfe4af9c eax:0xb70d981c ebx:0xbfe4af50 ecx:0xb70d97cc edx:0xb70d97c8 POLICY: controlled exit
*** PEASOUP: Add/Sub overflow detected at: 0x8055165 nextPC:0xf0001f30 esp:0xbfe4afec eax:0xbfe4ae30 ebx:0xbfe4afa0 ecx:0x3 edx:0xb70d97bc POLICY: controlled exit
