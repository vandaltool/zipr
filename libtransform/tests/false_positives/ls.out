bin
boot
cdrom
dev
etc
home
initrd.img
initrd.img.old
lib
lost+found
media
mnt
opt
proc
root
sbin
selinux
srv
sys
tmp
usr
var
vmlinuz
vmlinuz.old
